wp-content/themes/workingnyc/src/js/modules/search-archive.js

```diff
@@ -0,0 +1,243 @@
+'use strict';
```

Comment:
This is copied from modules/jobs-archive.js. This extends the Archive file in the WP Archive package and specifies the API endpoints to use, how to handle responses from the API endpoints, and some additional functions